doc/decompressor_errata.md

```diff
@@ -6,12 +6,29 @@ Each entry will contain:
 1. The last affected decompressor versions.
 2. The decompressor components affected.
 2. Whether the compressed frame could ever be produced by the reference compressor.
-3. An example frame.
+3. An example frame when it can be short enough to be displayed as hexadecimal
 4. A description of the bug.
 
 The document is in reverse chronological order, with the bugs that affect the most recent zstd decompressor versions listed first.
 
 
+Compressed block with a size of exactly 128 KB
+------------------------------------------------
+
+**Last affected version**: v1.5.2
+
+**Affected decompressor component(s)**: Library & CLI
+
+**Produced by the reference compressor**: No
+
```

Comment:
Can you please provide an example frame?

Since it is 128KB to produce it, maybe just add an example to https://github.com/facebook/zstd/tree/dev/tests/golden-decompression and provide a link here.

Adding the file there also automatically enables it as a test case.